spec/test_app/config/database.yml

```diff
@@ -4,22 +4,23 @@
 #   Ensure the SQLite 3 gem is defined in your Gemfile
 #   gem 'sqlite3'
 #
-default: &default
```

Comment:
was this not working? or do you just have a preference of explicitly stating the variables for each environment?

Reply:
This was not working because of some Psych stuff.